packages/perseus/src/strings.ts

```diff
@@ -1466,6 +1470,8 @@ export const mockStrings: PerseusStrings = {
     imageExploreButton: "Explore image",
     imageAlternativeTitle: "Explore image and description",
     imageDescriptionLabel: "Description",
+    imageZoomAriaLabel: "Zoom image.",
```

Comment:
for consistency, we don't normally add period

```suggestion
    imageZoomAriaLabel: "Zoom image",
```